hs_geo_raster_resource/receivers.py

```diff
@@ -140,6 +278,17 @@ def raster_pre_delete_file_from_resource_trigger(sender, **kwargs):
     res.metadata.create_element('BandInformation', name='Band_1', variableName='Unnamed', variableUnit='Unnamed',
                                 method='', comment='')
 
+    # delete all the files that is not the user selected file
+    for f in ResourceFile.objects.filter(object_id=res.id):
+        if f.resource_file.name != del_file.resource_file.name:
+            f.resource_file.delete()
```

Comment:
delete_resource_file() function send signal of pre_delete_file_from_resource(). It seems these two functions will call each other as a loop if I use delete_resource_file() in this signal function.